src/components/nav-logo/nav-logo.tsx

```diff
@@ -0,0 +1,96 @@
+import { Component, Element, EventEmitter, h, Host, Prop, Event } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-nav-logo",
+  styleUrl: "nav-logo.scss",
+  shadow: true
```

Comment:
Should these new components have delegatesFocus? Applies to all new components

Reply:
yup, these components need `delegatesFocus`.